src/main/java/com/exactpro/th2/readcsv/Main.java

```diff
@@ -275,11 +359,28 @@ private static Unit publishMessages(MessageRouter<RawMessageBatch> rawMessageBat
         return Unit.INSTANCE;
     }
 
+    @NotNull
+    private static Unit publishTransportMessages(MessageRouter<GroupBatch> transportBatchRouter, StreamId streamId, List<? extends RawMessage.Builder> builders, String bookName) {
+        try {
+            // messages are grouped by session aliases
+            String sessionGroup = builders.get(0).idBuilder().getSessionAlias();
```

Comment:
Shouldn't we check that builders variable has a least one element before accessing it by index?